app/observers/repository_observer.rb

```diff
@@ -5,9 +5,8 @@ def update_hook
     if object.saved_change_to_tracked?
       if object.tracked
         HookService.new.subscribe(object)
-        # TODO Patch until organization-repository relation exists
-        admin = AdminUser.all.first
-        GithubWorker.perform_async('FETCH_REPOS_PULL_REQUEST', owner_id: admin.id, repository_id: object.id)
+        GithubWorker.perform_async('FETCH_REPOS_PULL_REQUEST',
+                                   owner_id: object.organization.owner_id, repository_id: object.id)
```

Comment:
Esta perfecto en funcionalidades :) pero tengo algunos detalles de convención. 

Para parametros usa 1 identación
```ruby
GithubWorker.perform_async('FETCH_REPOS_PULL_REQUEST',
   owner_id: object.organization.owner_id, repository_id: object.id)
```

Lo otro, creo que el commit te quedo demasiado largo (y esta mal escrito subscribe). Podrías acortarlo por `user owner of organization in webhook and PR` o similar para que no se corte, con [amend](https://help.github.com/articles/changing-a-commit-message/).